automation/project/Dependencies.scala

```diff
@@ -13,7 +13,7 @@ object Dependencies {
   val workbenchGoogle2V = s"0.25-${workbenchLibsHash}"
   val workbenchModelV  = s"0.15-${workbenchLibsHash}"
   val workbenchMetricsV  = s"0.5-${workbenchLibsHash}"
-
+  val leonardoClient: ModuleID = "org.broadinstitute.dsde.workbench" %% "leonardo-client" % "1.3.5-178c33e")
```

Comment:
is this change necessary? I don't see the `leonardoClient` being used at all in the automation subproject.

Reply:
Removed